.github/workflows/node.js.yml

```diff
@@ -20,12 +19,11 @@ jobs:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v2
-      with:
-        node-version: ${{ matrix.node-version }}
-        cache: 'npm'
```

Comment:
Prettier is a shame but not worth fighting. Actual change is to drop the `cache: "npm"` line due to cryptic failures in GH actions. There's no cache I can delete via the UI and the dependencies are quick to install.

Reply:
Happy to drop the caching, but I wonder if reinstating the caching after running the action without it might fix it?